doc/api_reference.rst

```diff
@@ -7,83 +7,85 @@ API Reference
 _helpers
```

Comment:
Hazem, good work, however, maybe you should revert the documentation files that have been modified while you were editing your branch.
You may just copy and paste the files in the doc folder from the new-main into your branch and that should fix that issue.